docs/how_to_contribute.md

```diff
@@ -287,109 +326,119 @@ In the following table, you can find a short description of the fields of the `p
 
-    @attribute
-    def title(self) -> Optional[str]:
-        return self.precomputed.meta.get("og:title")
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.meta.get("og:title")
 ```
 
 Now, execute the example script from step 4 to validate your implementation.
 If the attributes are implemented correctly, they appear in the printout accordingly.
 
-``` console
+```console
 Fundus-Article:
-- Title: "Sarah Snook wasn't sold on 'Succession' at first. Now, she feels a 'sense of loss'"
-- Text:  "--missing plaintext--"  # TODO ADD THIS
-- URL:    https://www.latimes.com/entertainment-arts/tv/story/2023-03-19/sarah-snook-shiv-succession
-- From:   LosAngelesTimes (2023-03-20 19:25)
+- Title: "One hundred years at the Coliseum: Much more than a sports venue"
+- Text:  "Construction for the Los Angeles Coliseum was completed on May 1, 1923. Capacity
+          at the time: 75,000. The stadium was designed by architects John [...]"
+- URL:    https://www.latimes.com/sports/story/2023-06-26/100-years-los-angeles-coliseum-historical-events
+- From:   Los Angeles Times (2023-06-26 12:00)
 Fundus-Article:
-- Title: "Manhattan Beach mayor apologizes to Bruce's Beach families, unveils new city monument"
-- Text:  "--missing plaintext--"  # TODO ADD THIS
-- URL:    https://www.latimes.com/california/story/2023-03-18/la-me-bruces-beach-manhattan-beach-new-monument-ceremony
-- From:   LosAngelesTimes (2023-03-20 19:25)
+- Title: "Sparks back at .500: Five things to know about the team after win Sunday"
+- Text:  "Finally, the home crowd at Crypto.com Arena had something to cheer about.  After
+          dropping the first three games of their longest homestand of the [...]"
+- URL:    https://www.latimes.com/sports/sparks/story/2023-06-25/los-angeles-sparks-dallas-wings-wnba-game-analysis
+- From:   Los Angeles Times (2023-06-25 21:30)
 ```
 
-### 6. Writing Tests
+### 6. Generate unit tests
 
-Add a test case to `tests/resources/parser/test_data/<country_section>/` by compressing the HTML of an example article of your publisher to `<publisher_name>.html.gz`, e.g. `LosAngelesTimes.html.gz`.
-Next, specify asserted values your parser should extract from the example HTML in `<publisher_name>.json`, e.g. `LosAngelesTimes.json`.
-Currently, we only support tests for the `title`, `authors` and `topics` attributes.
+To finish your newly added publisher you should add unit tests for the parser.
+We recommend you do this with the provided [**script**](../scripts/generate_parser_test_files.py).
+First you should get known to the script.
+Just read through the manual provided with:
 
-A `LosAngelesTimes.json` may look like the following.
+````shell
+python -m scripts.generate_parser_test_files -h
+````
 
-```json
-{
-  "authors": [
-    "Luca Evans"
-  ],
-  "title": "Victims of Nashville school shooting honored in somber vigil"
-}
-```
+In our case, we would run something like this
```

Comment:
In my opinion is an example of "informal, personal" tone. I am not quite sure if this suits the type of content we intend to write. But since this is happening all the time and a subject of personal preference, we should discuss this in an issue.

Reply:
Maybe you could create an Issue about docs strings/tutorials best practice together with the comments in #239

Reply:
Alright, I will do this soon.

Reply:
I am fine with merging this as is, since once we decide, we will have to scan for instances in the whole repo anyway.